### ISSUE TITLE
Are you training the network with different data at each epoch?

### ISSUE THREAD

**Issue by lsaeuro** [QingyongHu/RandLA-Net#234]
Hi, thank you for your work! I am implementing the network with custom data and I am struggling in the dataloading.
I tried cutting squares but I ended up with different input sizes and forced to keep batch =1. 
I want to cut and save the input files as samples before the training - I have very huge point clouds and very sparse in some regions - so I was thinking on a strategy to have consistent, fixed size inputs.

Looking at [the code here](https://github.com/QingyongHu/RandLA-Net/blob/master/main_SemanticKITTI.py#L70)
 I was wandering if:
1. At each epoch the picked index is different so the training data are inconsistent between epochs or are you avoiding this effect with a manual seed?
2.  How can you be sure that you are covering the whole point clouds in training phase?
